### PR TITLE
Add alerts interval in create dialog and alert metadata page

### PR DESCRIPTION
**Pull request by AdityaHegde**
This PR adds Hour, Day, Week and Month as presets. Any non preset duration will be humanised similar to the time range labels.

Builds on top of #4033.

### CODE REVIEW

File: web-common/src/features/alerts/CreateAlertDialog.svelte [hunk 1]
```diff
@@ -104,7 +108,7 @@
           data: {
             options: {
               title: values.name,
-              intervalDuration: undefined, // TODO: this is the "for every" field I think?
+              intervalDuration: values.forEvery, // TODO: this is the "for every" field I think?
```
Comment (ericpgreen2) [inline web-common/src/features/alerts/CreateAlertDialog.svelte:111]: Can delete the TODO now

**Review by ericpgreen2 — COMMENTED** [review]
I'm not quite sure where it's coming from, but I'm hitting this 400 error on this branch, but not on `main`.

image

**Review by ericpgreen2 — APPROVED** [review]
Otherwise, this generally looks good, so I'll approve for you to merge at your discretion.

**Comment by AdityaHegde** [comment]
I was able to reproduce the issue on main. It is not related to this PR.

**Pull request merged**